.github/workflows/unit-tests.yml

```
@@ -47,6 +47,12 @@ jobs:
             continue_on_error: false
             job_label_php: 'current'
             job_label_wordpress: 'latest'
+          - php: '8.0'
+            wordpress: '6.2.0'
+            check_code_coverage: true
```

We only need to run the code-coverage checks (and Sonar) once, and that should happen for the default WP version:

```suggestion
            check_code_coverage: false
```

---

Thanks for the feedback @ethitter! Applying those changes.